### PR TITLE
Broadcast to Locations

### PR DESCRIPTION
**Pull request by gcapalbo**
This PR does two things:
- creates a new UX for broadcasts which uses crispy forms (I'll submit another PR to remove the old broadcasts UX)
- allows location(s) to be recipients of broadcasts

@kaapstorm code review buddy

broadcast_to_locations2
broadcast_to_locations1

### CODE REVIEW

File: corehq/apps/reminders/urls.py [hunk 1]
```diff
@@ -10,11 +10,19 @@
...
+    EditBroadcastView,
+    CopyBroadcastView,
 )
 
 urlpatterns = patterns('corehq.apps.reminders.views',
     url(r'^list/$', RemindersListView.as_view(), name=RemindersListView.urlname),
     url(r'^delete/(?P<handler_id>[\w-]+)/$', 'delete_reminder', name='delete_reminder'),
+    url(r'^broadcasts/$', BroadcastListView.as_view(), name=BroadcastListView.urlname),
+    url(r'^broadcasts/add/$', CreateBroadcastView.as_view(), name=CreateBroadcastView.urlname),
+    url(r'^broadcasts/edit/(?P<broadcast_id>[\w-]+)/$', EditBroadcastView.as_view(), name=EditBroadcastView.urlname),
```
Comment (dimagimon) [inline corehq/apps/reminders/urls.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/reminders/urls.py [hunk 2]
```diff
@@ -10,11 +10,19 @@
...
+    url(r'^broadcasts/copy/(?P<broadcast_id>[\w-]+)/$', CopyBroadcastView.as_view(), name=CopyBroadcastView.urlname),
```
Comment (dimagimon) [inline corehq/apps/reminders/urls.py]: E501 line too long (117 > 115 characters)

**Comment by kaapstorm** [comment]
:+1: Looks good to me.

**Comment by gcapalbo** [comment]
@kaapstorm ok, this is good to go now

**Pull request merged**